<?xml version="1.0"?>
<launch>

    <node pkg="explore_large_map" type="map_entry" name="map_entry_node" output="screen" >
        <param name="map_width" type="double" value="100"/>
        <param name="map_height" type="double" value="80" />
        <param name="local_map_resolution" type="double" value="0.2" />
        <param name="local_map_frame_name" value="base_link" />
    </node>

    <node pkg="explore_large_map" type="explore_large_map_node" name="explore_large_map_node" output="screen" >
        <param name="map_width" type="double" value="100"/>
        <param name="map_height" type="double" value="100" />
        <param name="map_resolution" type="double" value="0.2" />
        <param name="unknown_value" type="int" value="0" />
    </node>


    <node pkg="rviz" type="rviz" name="rviz_explore_map" args="-d $(find explore_large_map)/cfg/explore_map_viz.rviz"/>

</launch>
